frontend/src/api/network/inferenceServices.ts

```diff
@@ -54,13 +55,41 @@ const assembleInferenceService = (
   };
 };
 
-export const listInferenceService = (namespace?: string): Promise<InferenceServiceKind[]> => {
+export const listInferenceService = (
+  namespace?: string,
+  labelSelector?: string,
+): Promise<InferenceServiceKind[]> => {
+  const queryOptions = {
+    ...(namespace && { ns: namespace }),
+    ...(labelSelector && { queryParams: { labelSelector } }),
+  };
   return k8sListResource<InferenceServiceKind>({
     model: InferenceServiceModel,
-    ...(namespace && { queryOptions: { ns: namespace } }),
+    queryOptions,
   }).then((listResource) => listResource.items);
 };
 
+export const listScopedInferenceService = (
+  labelSelector?: string,
+): Promise<InferenceServiceKind[]> => {
+  return getModelServingProjects().then((projects) => {
+    return Promise.all(
+      projects.map((project) => listInferenceService(project.metadata.name, labelSelector)),
+    ).then((listInferenceService) => _.uniq(_.flatten(listInferenceService)));
```

Comment:
As I noted in a DM, we should probably use [_.uniqBy](https://lodash.com/docs/4.17.15#uniqBy) instead. Not sure we should try to compare a k8s object like this.